LiskHQ/lisk-sdk
# Support all tokens when defined in genesis assets

### What was the problem?

This PR resolves #8761 

### How was it solved?

- Call `supportAll` when genesis assets have `chainID === ''` and empty `supportedTokenIDs`
- Update tests to test support all feature
- Update example app

### How was it tested?

- Update genesis block with genesis assets as 
```json
"supportedTokensSubstore": [
					{
						"chainID": "",
						"supportedTokenIDs": []
					}
				]
```
- Run application and call `token_getSupportedTokens` and it should return below
```json
"supportedTokens": [
            "*"
        ]
```
